docs/news.rst

```diff
@@ -14,6 +14,7 @@ Python 3.3.0 (unreleased)
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
 - Drop support for Python 3.8.
+- Add support for binary hstore (:ticket:`#1030`).
```

Comment:
The version in which this will be released is 3.2.7. If you can please pull from master and rebase you will see an entry for that version.